docs/docs/guide/usingLyra.md

```diff
@@ -51,18 +51,23 @@ if (isConnected()) {
 }
```

Comment:
docs updated because lyra-api returns strings for these functions now instead of an obj